src/Http/Controllers/RequestController.php

```diff
@@ -9,15 +9,27 @@
 class RequestController extends Controller
 {
     /**
-     * Stores action and visit counts
+     * Saves action/visit count from cron job
      *
-     * @param StoreRequest $request containing the json object
+     * @api {post} /kpis
+     * @apiPermission token-verifyed
+     * @apiDescription Saves actions/visits of a serverlog file
```

Comment:
```suggestion
     * @apiPermission kpi-token
     * @apiDescription Save actions/visits KPIs
     * @apiParam {Number} visits The number of GET requests to `/` (without bots).
     * @apiParam {Number} actions The number of PUT/POST/DELETE requests (without bots and haertbeat).
```

The `apiPermission` value needs to be defined somewhere. You can do this by pasting this block above the controller method:

```php
/**
 * @apiDefine kpi-tpken KPI Token
 * The request must provide the token configured in the KPI module.
 */
```